`./gradlew build` comes errors

when i use the `./gradlew build` command, some errors come:
```
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not resolve org.ethereum:solcJ-all:0.4.25.
     Required by:
         project : > org.fisco-bcos:web3sdk:2.2.0
      > Could not resolve org.ethereum:solcJ-all:0.4.25.
         > Could not get resource 'https://dl.bintray.com/ethereum/maven/org/ethereum/solcJ-all/0.4.25/solcJ-all-0.4.25.pom'.
            > Could not GET 'https://dl.bintray.com/ethereum/maven/org/ethereum/solcJ-all/0.4.25/solcJ-all-0.4.25.pom'. Received status code 403 from server: Forbidden

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

```

--- a/build.gradle
+++ b/build.gradle
@@ -29,15 +29,15 @@ verifyGoogleJavaFormat {
 repositories {
     jcenter()
     maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
-    maven { url 'https://dl.bintray.com/ethereum/maven/' }
 }
 
 // In this section you declare the dependencies for your production and test code
 dependencies {
     compile 'org.slf4j:slf4j-log4j12:1.7.25'
-    compile 'org.fisco-bcos:web3sdk:2.2.0'
+    compile 'org.fisco-bcos:web3sdk:2.6.4'
     compile 'com.fasterxml.jackson.core:jackson-databind:2.8.8.1'
     compile 'com.google.guava:guava:19.0'
+    implementation group: 'com.alibaba', name: 'fastjson', version: '1.2.7'
 }
 
 jar {

